CHANGELOG.md

```diff
@@ -0,0 +1,90 @@
+# go-textile changelog
```

Comment:
So I think that we can actually use https://github.com/textileio/go-textile/releases as the changelog. And just link there. There is no reason why it needs to be a `CHANGELOG.md` or `HISTORY.md` file specifically (a convention older than github releases).

Reply:
Cool, I'm just copying go-ipfs here, but agreed, it does feels redundant... _unless_ we can hook into this for the release automation - pull the notes from a `CHANGELOG.md` file and dump into the release notes.